utils/gitlog2changelog.py

```diff
@@ -15,7 +16,20 @@
     rev_range = "%s..HEAD" % base
 
 # Execute git log with the desired command line options.
-fin = os.popen("git log --summary --stat --no-merges --date=short %s" % rev_range, "r")
+# uses subproccess instead, more secure.
+command =  ["git", "log", "--summary", "--stat", "--no-merges", "--date=short", rev_range]
+
+fin = subprocess.Popen(command, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+output, error = process.communicate()
+
+# Check for errors
+if process.returncode != 0:
+    # Handle error
+    print("Error:", error.decode("utf-8"))
+else:
+    # Process the output
+    print("Output:", output.decode("utf-8"))
```

Comment:
Too generic, more specific handling is needed here. See existing code below.

Reply:
this was removed by @kpolchow in a previous commit